### DISCUSSION TITLE
Submit Tulipa for the TNO-Sigrid pilot?

### DISCUSSION THREAD

**Discussion by clizbe** [TulipaEnergy/TulipaEnergyModel.jl#613] [Ideas]
The TNO Coding Guild is asking for programming projects to demo this software and see if it's useful to TNO:
https://docs.sigrid-says.com/

Shall we submit Tulipa as a candidate? (Mostly asking eScience if you think it adds value for us.)

@suvayu  @abelsiqueira

**Comment by abelsiqueira** [comment]
I think it surely adds value, if they can check Julia code. I heard a talk about another company that does a similar job and they were evaluating Julia and there were a lot of missing tooling. It would be great to know it there is an update on that.

PS. I looked around and found their supported technologies: https://docs.sigrid-says.com/reference/technology-support.html
Julia is not included. :(

**Reply by clizbe** [reply]
Ah bummer - thanks for checking! :)
